tests/adapters/switches/brocade_test.py

```diff
@@ -207,6 +207,16 @@ def test_get_vlans(self):
         assert_that(str(vlan201.dhcp_relay_servers[0]), equal_to('10.10.10.1'))
         assert_that(str(vlan201.dhcp_relay_servers[1]), equal_to('10.10.10.2'))
 
+        assert_that(vlan1.tagged_interfaces, equal_to([]))
+        assert_that(vlan201.tagged_interfaces, equal_to(["ethe 1/1"]))
+        assert_that(vlan2222.tagged_interfaces, equal_to(["ethe 1/1"]))
+        assert_that(vlan3333.tagged_interfaces, equal_to([]))
+
+        assert_that(vlan1.interfaces, equal_to(["ethe 1/1", "ethe 1/20", "ethe 1/21", "ethe 1/22"]))
```

Comment:
This seems the opposite of reality, after some testing on the brocade it would seem the vlan 1 is untagged by default on every interfaces unless specified

The show run says "no untagged..." which means these interfaces are the only ones not untagged on this vlan